### DISCUSSION TITLE
Problem with GitHub Student Pack access

### DISCUSSION THREAD

**Discussion by vincenttjia** [community/community#17702] [GitHub Education] [answered]
A few days ago I was asked to re-verify your academic affiliation, which I did and was approved
<img width="1728" alt="Screen Shot 2022-05-31 at 20 38 55" src="https://user-images.githubusercontent.com/48272053/171189705-42525f05-3847-4eda-8c19-7507c0685418.png">

<img width="1728" alt="Screen Shot 2022-05-31 at 20 40 45" src="https://user-images.githubusercontent.com/48272053/171189790-cb119324-9f94-4105-add9-6bc786877fe5.png">
But when I tried to access the student pack it still says that I need to re-verify

What should I do?

**Comment by vincenttjia** [comment]
Also if anyone is asking, I have submitted a ticket with no human reply so far

https://support.github.com/ticket/personal/0/1643426

**Comment by vincenttjia** [comment, marked as answer]
Nevermind I think it just need some time to propagate since I'm getting the access now. 
Or thanks to anyone who sees my ticket or discussion and help me resolve this issue.

Consider this issue solved

Thank you

**Comment by EPSM2022** [comment]
I will try again with different printed proof.  If it still fails, I am done.  I am wanting the discounts and easier ways to learn, but this process is a huge failure regarding user experience.  It is as if they want to tease the benefits and then not let people in.
